### PR TITLE
Satisfy pylama for log and some misc. files

### PR DESCRIPTION
**Pull request by pagrubel**
I had to ignore moving the import bee stuff; if one of you can change that I'm all for it.

### CODE REVIEW

File: src/beeflow/tests/test_tm.py [hunk 1]
```diff
@@ -73,22 +73,22 @@ def test_remove_task(flask_client, mocker):  # noqa
     """Test cancelling a workflow and removing tasks."""
     # Add a few tasks
     beeflow.task_manager.job_queue.append({2: {'name': 'task1',
-                                                            'job_id': 1,
```
Comment (jtronge) [inline src/beeflow/tests/test_tm.py:76]: It complains about the order of the imports in this file.

**Review by jtronge — COMMENTED** [review]
It has a few warnings in `src/beeflow/tests/test_tm.py`. When I ran `pylama src/beeflow/common/worker/` it also gave some warnings for `src/beeflow/common/worker/worker.py`, but those can probably be done in a different PR.

File: src/beeflow/tests/test_tm.py (hunk 1 above)
Comment (pagrubel) [reply src/beeflow/tests/test_tm.py:76]: I thought I fixed those... I'll look again

Comment (pagrubel) [reply src/beeflow/tests/test_tm.py:76]: Latest commit should fix those

**Review by jtronge — APPROVED** [review]

**Review by Boogie3D — APPROVED** [review]

**Pull request merged**